charts/dgraph/templates/alpha/ingress.yaml

```diff
@@ -1,3 +1,22 @@
+{{- /* Generate ingress path */}}
+{{- define "path" -}}
+  {{- $path := "/" -}}
+  {{- if .Values.global.ingress.ingressClassName -}}
+    {{- if eq .Values.global.ingress.ingressClassName "gce" "alb" "nsx" }}
```

Comment:
can you add comment for this conditional

Reply:
The `path` varies depending on the ingress controller.  Some ingress controllers require `/`, while others require `/*`.  

Ingress of `gce` (GLBC), `alb` (ALB), and `nsx` (VMWare) will require `/*`, while others will require `/`.